.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
```

Comment:
this should have a more descriptive name

Reply:
Changed to Light House CI this make more sense.